pkg/rewards/8_goldOperatorODRewardAmounts.go

```diff
@@ -0,0 +1,75 @@
+package rewards
+
+import (
+	"github.com/Layr-Labs/sidecar/pkg/rewardsUtils"
+	"go.uber.org/zap"
+)
+
+const _8_goldOperatorODRewardAmountsQuery = `
+create table {{.destTableName}} as
+-- Get the rows where operators have registered for the AVS
+WITH reward_snapshot_operators as (
+  SELECT
+    ap.reward_hash,
+    ap.snapshot as snapshot,
+    ap.token,
+    ap.tokens_per_day,
+    ap.tokens_per_day_decimal,
+    ap.avs as avs,
+    ap.operator as operator,
+    ap.strategy,
+    ap.multiplier,
+    ap.reward_submission_date
+  FROM {{.activeRewardsTable}} ap
+  JOIN operator_avs_registration_snapshots oar
+  ON ap.avs = oar.avs and ap.snapshot = oar.snapshot and ap.operator = oar.operator
+),
+-- Dedupe the operator tokens across strategies for each (operator, reward hash, snapshot) since the above result is a flattened out operator-directed reward submission across strategies.
+distinct_operators AS (
+  SELECT *
+  FROM (
+	  SELECT *,
+	    -- We can use an arbitrary order here since the avs_tokens is the same for each (operator, strategy, hash, snapshot)
+        -- We use strategy ASC for better debuggability
+		ROW_NUMBER() OVER (PARTITION BY reward_hash, snapshot, operator ORDER BY strategy ASC) as rn
+	  FROM reward_snapshot_operators
+  ) t
+  WHERE rn = 1
+),
+-- Calculate the tokens for each operator with dynamic split logic
+-- If no split is found, default to 1000 (10%)
```

Comment:
Why aren't we getting the operators that are registered to the AVS in this query?

Reply:
We are implicitly getting this in the next query through `operator_avs_strategy_snapshots` (which checks Strats of operators registered to the AVS), but would be much clearer if all the operators that were actually going to be paid were in the final output of this table

Reply:
It is right? The `reward_snapshot_operators` CTE above this joins with `operator_avs_registration_snapshots`

Reply:
Completely missed this in first pass, we're good